### DISCUSSION TITLE
Socket.io와 WebSocket의 차이를 설명해주세요.

### DISCUSSION THREAD

**Discussion by eatingbug** [Team-Psyduck/tech-sharing#78] [General]
[2293](https://www.acmicpc.net/problem/2293)

**Comment by SionBackEnd** [comment]
WebSocket은 양방향 소통을 위한 프로토콜로 서로 다른 컴퓨터끼리 소통하기 위한 약속입니다. 
Socket.io는 양방향 통신을 하기 위해 웹소켓 기술을 활용하는 라이브러리 입니다. 

Websocket은 이벤트를 단순히 듣고, 보내는 것만 가능하지만, Socket.io는 방 개념을 이용해 일부 클라이언트에게만 데이터를 전송하는 브로드캐스팅이 가능합니다. 

[참고](https://www.peterkimzz.com/websocket-vs-socket-io/)
